lib/sdnc

```diff
@@ -14,6 +14,7 @@ function build_sdnc_images {
 
 # install_sdnc() -  Download and install SDNC services from source code
 function install_sdnc {
+    install_unzip
```

Comment:
I was thinking that won't be necessary to create a new function for this, maybe only adding `install_package unzip` here will be enough.  The criteria behind that should be those functions where doesn't follow a standard installation.